src/envo/modules/chemical_concentration.csv

```diff
@@ -1,31 +1,88 @@
+ENVO:3200009,,CHEBI:30145,lithium,ENVO:04000007,lake water
+ENVO:3200010,,CHEBI:27376,methanesulfonic acid,ENVO:04000007,lake water
+ENVO:3200011,,CHEBI:28685,molybdenum,ENVO:04000007,lake water
+ENVO:3200012,,CHEBI:33372,neodymium,ENVO:04000007,lake water
+ENVO:3200013,,CHEBI:33344,niobium,ENVO:04000007,lake water
+ENVO:3200014,,CHEBI:49828,praseodymium,ENVO:04000007,lake water
+ENVO:3200015,,CHEBI:33374,samarium,ENVO:04000007,lake water
+ENVO:3200016,,CHEBI:30512,silver,ENVO:04000007,lake water
+ENVO:3200017,,CHEBI:30440,thallium,ENVO:04000007,lake water
+ENVO:3200018,,CHEBI:27007,tin,ENVO:04000007,lake water
+ENVO:3200019,,CHEBI:33341,titanium,ENVO:04000007,lake water
+ENVO:3200020,,CHEBI:33331,yttrium,ENVO:04000007,lake water
+ENVO:3200021,,CHEBI:22927,bromine,ENVO:01000277,water ice
+ENVO:3200022,,CHEBI:33369,cerium,ENVO:01000277,water ice
+ENVO:3200023,,CHEBI:27638,cobalt,ENVO:01000277,water ice
+ENVO:3200024,,CHEBI:33377,dysprosium,ENVO:01000277,water ice
+ENVO:3200025,,CHEBI:32999,europium,ENVO:01000277,water ice
+ENVO:3200026,,CHEBI:33375,gadolinium,ENVO:01000277,water ice
+ENVO:3200027,,CHEBI:33336,lanthanum,ENVO:01000277,water ice
+ENVO:3200028,,CHEBI:30145,lithium,ENVO:01000277,water ice
+ENVO:3200029,,CHEBI:27376,methanesulfonic acid,ENVO:01000277,water ice
+ENVO:3200030,,CHEBI:28685,molybdenum,ENVO:01000277,water ice
+ENVO:3200031,,CHEBI:33372,neodymium,ENVO:01000277,water ice
+ENVO:3200032,,CHEBI:33344,niobium,ENVO:01000277,water ice
+ENVO:3200033,,CHEBI:49828,praseodymium,ENVO:01000277,water ice
+ENVO:3200034,,CHEBI:33374,samarium,ENVO:01000277,water ice
+ENVO:3200035,,CHEBI:30512,silver,ENVO:01000277,water ice
+ENVO:3200036,,CHEBI:30440,thallium,ENVO:01000277,water ice
+ENVO:3200037,,CHEBI:27007,tin,ENVO:01000277,water ice
+ENVO:3200038,,CHEBI:33341,titanium,ENVO:01000277,water ice
+ENVO:3200039,,CHEBI:33331,yttrium,ENVO:01000277,water ice
+ENVO:3200040,,CHEBI:22927,bromine,ENVO:01000406,snow
+ENVO:3200041,,CHEBI:33369,cerium,ENVO:01000406,snow
+ENVO:3200042,,CHEBI:27638,cobalt,ENVO:01000406,snow
+ENVO:3200043,,CHEBI:33377,dysprosium,ENVO:01000406,snow
+ENVO:3200044,,CHEBI:32999,europium,ENVO:01000406,snow
+ENVO:3200045,,CHEBI:33375,gadolinium,ENVO:01000406,snow
+ENVO:3200046,,CHEBI:33336,lanthanum,ENVO:01000406,snow
+ENVO:3200047,,CHEBI:30145,lithium,ENVO:01000406,snow
+ENVO:3200048,,CHEBI:27376,methanesulfonic acid,ENVO:01000406,snow
+ENVO:3200049,,CHEBI:28685,molybdenum,ENVO:01000406,snow
+ENVO:3200050,,CHEBI:33372,neodymium,ENVO:01000406,snow
+ENVO:3200051,,CHEBI:33344,niobium,ENVO:01000406,snow
+ENVO:3200052,,CHEBI:49828,praseodymium,ENVO:01000406,snow
+ENVO:3200053,,CHEBI:33374,samarium,ENVO:01000406,snow
+ENVO:3200054,,CHEBI:30512,silver,ENVO:01000406,snow
+ENVO:3200055,,CHEBI:30440,thallium,ENVO:01000406,snow
+ENVO:3200056,,CHEBI:27007,tin,ENVO:01000406,snow
+ENVO:3200057,,CHEBI:33341,titanium,ENVO:01000406,snow
+ENVO:3200058,,CHEBI:33331,yttrium,ENVO:01000406,snow
```

Comment:
@kaiiam @stevenchong 
The lack of a newline at the EOF can sometimes cause issues. Did you validate that all terms added to this CSV are created in your local copies of the editor's OWL file?

Reply:
> Please confirm that you both see all the terms added here in the editor's version of the OWL file.

I don't see any of the new concentration terms in my copy of envo-edit.owl

Reply:
When I make the chemical_concentrations.owl file it works fine without the newline.

Reply:
@kaiiam thanks for the confirmation. All good.